site/documentation/static/api/device-registry-v1.yaml

```diff
@@ -629,12 +650,31 @@ components:
                  "hash-function":
                     type: string
                     example: sha-512
+                    description: The name of the hash function used to create the password hash (defined in 'pwd-hash' property).
+                                 If password is defined using 'pwd-plain' property, this value will be ignored by the device registry.
```

Comment:
I think it sounds better with added articles:
````
(defined in the 'pwd-hash' property).
If the password is defined using a 'pwd-plain' property, this value will be ignored by the device registry.
````
Same in the two description blocks below.